### PR TITLE
add more splitters; improve debugger UI layout for smaller widths

### PR DESCRIPTION
**Pull request by devoncarew**
- add a vertical splitter to the debugger page
- add a vertical splitter to the logging page
- improve the layout behavior of the debugger controls for smaller page widths
- unrelated, but remove extra whitespace in our in-line error component

@jacob314 @kenzieschmoll

### CODE REVIEW

**Comment by devoncarew** [comment]
screen shot 2019-01-07 at 4 58 22 pm

File: lib/debugger/debugger.dart [hunk 1]
```diff
@@ -170,11 +168,21 @@ class DebuggerScreen extends Screen {
...
+    // configure the navigation / editor splitter
     split.flexSplit(
-      panels,
-      gutterSize: 12,
-      sizes: [25, 75],
-      minSize: [150, 200],
+      navEditorPanels,
+      gutterSize: defaultSplitterWidth,
+      sizes: [22, 78],
+      minSize: [200, 600],
```
Comment (jacob314) [inline lib/debugger/debugger.dart]: strangely, the chrome-devtools allow very small min-sizes for the text editor and stack frame windows. However, I agree that these more practical minSizes are worth a try at least until the UI actually looks alright with smaller widths.

File: lib/debugger/debugger.dart [hunk 2]
```diff
@@ -170,11 +168,21 @@ class DebuggerScreen extends Screen {
...
+    );
+
+    // configure the editor / console splitter
+    split.flexSplit(
```
Comment (jacob314) [inline lib/debugger/debugger.dart]: excellent!

File: lib/debugger/debugger.dart (hunk 1 above)
Comment (devoncarew) [reply lib/debugger/debugger.dart]: Yeah, happy to tweak these going forward, esp. if we find the min sizes cause issues for users.

**Pull request merged**